example/lib/polygon_annotations_example.dart

```diff
@@ -34,12 +35,15 @@ class PolygonAnnotationExampleState extends State<PolygonAnnotationExample> {
       polygonAnnotationManager = value;
       createOneAnnotation();
       var options = <PolygonAnnotationOptions>[];
-      for (var i = 0; i < 2; i++) {
+      for (var i = 0; i < 500; i++) {
```

Comment:
Similar to polyline, reduce the number so we can see a bit of the actual map 

<img width="1206" height="2622" alt="Image" src="https://github.com/user-attachments/assets/2bcf83f6-e263-4a74-85fe-d402e7058280" />